iroha_actor/src/lib.rs

```diff
@@ -445,19 +445,20 @@ impl<A: Actor> Context<A> {
 
     /// Notifies actor with items from stream
     pub fn notify_with<M, S>(&self, mut stream: S)
-    where
-        M: Message<Result = ()> + Send + 'static,
-        S: Stream<Item = M> + Unpin + Send + 'static,
-        A: Handler<M>,
+        where
+            M: Message<Result = ()> + Send + 'static,
+            S: Stream<Item = M> + Send + 'static,
+            A: Handler<M>,
     {
         let addr = self.addr();
         drop(task::spawn(
             async move {
+                let mut stream = unsafe { std::pin::Pin::new_unchecked(&mut stream) };
```

Comment:
i guess better to use `futures::pin_mut` macro. It is available in futures, not only in futures-util.

Reply:
I think it's your code, actually ;))

Reply:
Yeah, its mine, but it was quick and dirty check :)

At the time of call i though that it would be faster to expand macro by hand than import futures_util :)